playbooks/common/openshift-master/config.yml

```diff
@@ -5,6 +5,10 @@
     t_oo_option_master_debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') }}"
 
   pre_tasks:
+  - name: Fail early if Atomic and old version of Docker
+    include: ../../../roles/openshift_common/tasks/atomic_docker_check.yml
```

Comment:
What about using ``roles/openshift_{master,node}/tasks/main.yml``? It looks like the variables are defined there.